src/main/java/com/example/controller/LegendController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.controller;
+
+import javafx.fxml.FXML;
+import net.rgielen.fxweaver.core.FxmlView;
+import org.springframework.stereotype.Component;
+
+@Component
+@FxmlView("/view/statisticsView.fxml")
+public class LegendController {
+
+    @FXML
+    public void initialize() {
```

Comment:
This method doesn't do anything, so probably can be removed

Reply:
Controller do not work without this method, however it is unneccessary, so deleted LegendController